react_client/vite.config.js

```diff
@@ -5,6 +5,14 @@ import react from '@vitejs/plugin-react'
 export default defineConfig({
   plugins: [react()],
   build: {
-    outDir: "../dist"
+    outDir: "../public"
+  },
+  server: {
+    proxy: {
+      '/api': {
+        target: 'http://localhost:5523',
```

Comment:
this is how the vite app now speaks to the espress app in dev mode, i've setup vercel to unwrap this and serve it in production ready mode too via the build step and the vite proxy config add. please see vercel.json for more